serde_test/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "serde_test"
+version = "0.1.0"
```

Comment:
Since people are actually meant to use this crate, maybe we should sync versions with serde. Otherwise this becomes a piston-like mess